### ISSUE TITLE
Add support for angular 9

### ISSUE THREAD

**Issue by AhmedBenyahia** [qontu/ngx-inline-editor#157]
Starting from  angular 9, Renderer has been renamed to Renderer2. 
After upgrading to angular 9 I am getting this error: 

ERROR in node_modules/@qontu/ngx-inline-editor/inputs/input-base.d.ts:2:10 - error TS2724: Module '"../../../@angular/core/core"' has no exported member 'Renderer'. Did you mean 'Renderer2'?

**Comment by ZiedMayel** [comment]
Use Renderer2 instead of Renderer
________________________________
Von: Ahmed Benyahia <notifications@github.com>
Gesendet: Montag, 17. August 2020 15:20
An: qontu/ngx-inline-editor <ngx-inline-editor@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [qontu/ngx-inline-editor] Add support for angular 9 (#157)


Starting from angular 9, Renderer has been renamed to Renderer2.
After upgrading to angular 9 I am getting this error:

ERROR in node_modules/@qontu/ngx-inline-editor/inputs/input-base.d.ts:2:10 - error TS2724: Module '"../../../@angular/core/core"' has no exported member 'Renderer'. Did you mean 'Renderer2'?

**Comment by AhmedBenyahia** [comment]
It work as a workaround but it need a proper fix.
It's not a good habit to modify the files in node_module manually

**Comment by ZiedMayel** [comment]
I agree, I mean you have to fork from the repository and fix that. use npm link to use it locally or add it to your project as shared library. Unfortunately the project is not maintained
________________________________
Von: Ahmed Benyahia <notifications@github.com>
Gesendet: Montag, 17. August 2020 15:44
An: qontu/ngx-inline-editor <ngx-inline-editor@noreply.github.com>
Cc: ZiedMayel <z.mayel@hotmail.com>; Comment <comment@noreply.github.com>
Betreff: Re: [qontu/ngx-inline-editor] Add support for angular 9 (#157)


It work as a workaround but it need a proper fix.
It's not a good habit to modify the files in node_module manually

**Comment by dogetomoon2021** [comment]
@ZiedMayel Can you please elaborate on the steps to follow?

**Comment by ZiedMayel** [comment]
@dogetomoon2021 

> @ZiedMayel Can you please elaborate on the steps to follow?


What do you need exactly? Can you please tell me what do you mean with "elaborate on the steps"?

**Comment by dogetomoon2021** [comment]
@ZiedMayel I want to use this lib. with angular 10 or are you aware of any alternative to this lib?

**Comment by AhmedBenyahia** [comment]
I suggest u create your own lib. I have created a simple component to replace  the old one but it's not in a package, you will need to include it manually in the project: https://github.com/SlaySayto/s-inline-editor
